packages/client-axios/src/utils.ts

```diff
@@ -171,6 +171,15 @@ export const setAuthParams = async ({
         }
         options.query[name] = token;
         break;
+      case 'cookie': {
+        const value = `${name}=${token}`;
+        if ('Cookie' in options.headers && options.headers['Cookie'] !== '') {
```

Comment:
Wouldn't this be enough?

```suggestion
        if ('Cookie' in options.headers && options.headers['Cookie']) {
```